test/functional/kubernetes/kubernetes_test.go

```diff
@@ -48,6 +48,7 @@ import (
 )
 
 func Test_TutorialApplication_KubernetesManifests(t *testing.T) {
+	t.Skip("Skipping for now, will reenable before merging.")
```

Comment:
Could you please summarize the changes for the PR in the description.. the current seems outdated?

Also, curious why does this test need to be skipped for this PR to be merged?

Reply:
I am still testing some changes here. I guess I should mark it as Draft for now so that I can get reviews later.

Reply:
This was also reverted to the original.